examples/traverse_graph.rs

```diff
@@ -0,0 +1,135 @@
+use dot_vox::{DotVoxData, Model, Rotation, SceneNode};
+use glam::Vec3;
+
+fn iterate_vox_tree<F: FnMut(&Model, &Vec3, &Rotation) -> ()>(vox_tree: &DotVoxData, mut fun: F) {
+    // An iterator for each level of depth, during depth-first search:
```

Comment:
Iterator has a specific meaning in Rust. A better description might be
```suggestion
// The stack for maintaining iteration state.
```